tests/melt_material_4.cc

```diff
@@ -0,0 +1,252 @@
+#include <aspect/material_model/interface.h>
+#include <aspect/velocity_boundary_conditions/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+#include <aspect/melt.h>
+#include <aspect/fluid_pressure_boundary_conditions/interface.h>
+
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+using namespace dealii;
+
+
+namespace aspect
+{
+  template <int dim>
+  class MeltMaterial:
+    public MaterialModel::Interface<dim>, public ::aspect::SimulatorAccess<dim>
+  {
+      virtual bool
+      viscosity_depends_on (const MaterialModel::NonlinearDependence::Dependence dependence) const
```

Comment:
same. Probably I need to sit down and make a PR for every test.cc that still uses them. Somehow I thought I did that long ago already, maybe you branched off before that happened? Yep just checked, your tests are the only ones that still use them. Please remove them from all test .cc files. This should have no influence on the solution.